actionkit/api/action.py

```diff
@@ -0,0 +1,22 @@
+import json
+import re
+import requests
+
+from actionkit.api import base
+
+class AKActionAPI(base.ActionKitAPI):
+
+    def create_action(self, akid, page, fields=None):
+        data = {
+            'akid': akid,
+            'page': page
+        }
+        if fields is not None:
```

Comment:
Saw this after I merged, so added in https://github.com/MoveOnOrg/python-actionkit/commit/55e0417e5f675bccdd0f361a6d1923dedfcba264